pkg/resourceinterpreter/configurableinterpreter/luavm/lua.go

```diff
@@ -423,6 +423,12 @@ func decodeValue(L *lua.LState, value interface{}) lua.LValue {
 			arr.Append(decodeValue(L, item))
 		}
 		return arr
+	case []workv1alpha2.AggregatedStatusItem:
+		arr := L.CreateTable(len(converted), 0)
+		for _, item := range converted {
+			arr.Append(decodeValue(L, item))
+		}
+		return arr
```

Comment:
I am wonder this code working? I didn't find where to decode `AggregatedStatusItem`

Reply:
I mean this code can't decode `AggregatedStatusItem`. I see no case to decode it.

Reply:
I'm fixing it in https://github.com/karmada-io/karmada/pull/2803

Reply:
@ikaven1024 In my opinion if we find something wrong, we should talk about it here, starting a new PR wouldn't help to accelerate the progress, because this will turn our focus to another PR.